cmd/vulkan_sample/frag.h

```diff
@@ -0,0 +1,67 @@
+{0x07230203,0x00010000,0x000d0007,0x00000026,
```

Comment:
Missing copyright header

Also, I would add a small comment to say this is binary SPIR-V, generated from cmd/vulkan_sample/shader.frag (which I assume is the case)